script/readyhelper.js

```diff
@@ -54,11 +54,11 @@ define(
           return currentTime >= 0;
         }
 
-        if (isValidSeekableRange(seekableRange)) {
-          return currentTime >= seekableRange.start &&
```

Comment:
Does this fix a bug outside of this PR or should this check really be inside `isValidSeekableRange`?

Reply:
The checking of currentTime being within the seekable range was breaking things on occasions for this new basicStrategy. My understanding is this (currentTime lies within the seekableRange) isn't a spec requirement, so we decided to relax that rule here (will apply to all strategies of course).